sympy/stats/crv_types.py

```diff
@@ -108,7 +108,7 @@
 
 def ContinuousRV(symbol, density, set=Interval(-oo, oo)):
     """
-    Creates a Continuous Random Variable given the following:
+    Create a Continuous Random Variable given the following:
```

Comment:
I've seen that an s is used in some documentation, but I do not have any strong opinions. However, maybe you would like to change to "continuous random variable" as in most cases below.

Reply:
This was changed a couple of weeks ago. I think it is proper to return the old form recommended by PEP 257:
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".